diff --git a/EgammaAnalysis/ElectronTools/plugins/RegressionEnergyPatElectronProducer.cc b/EgammaAnalysis/ElectronTools/plugins/RegressionEnergyPatElectronProducer.cc
index 0ddae78..86819f3 100644
--- a/EgammaAnalysis/ElectronTools/plugins/RegressionEnergyPatElectronProducer.cc
+++ b/EgammaAnalysis/ElectronTools/plugins/RegressionEnergyPatElectronProducer.cc
@@ -67,8 +67,8 @@ RegressionEnergyPatElectronProducer::RegressionEnergyPatElectronProducer( const
   regressionEvaluator_->initialize(regressionInputFile_.c_str(),type);
 
   if(produceValueMaps_) {
-    produces<edm::ValueMap<double> >(nameEnergyReg_);
-    produces<edm::ValueMap<double> >(nameEnergyErrorReg_);
+    produces<edm::ValueMap<float> >(nameEnergyReg_);
+    produces<edm::ValueMap<float> >(nameEnergyErrorReg_);
   }
 
 
@@ -159,16 +159,16 @@ void RegressionEnergyPatElectronProducer::produce( edm::Event & event, const edm
   std::auto_ptr<ElectronCollection> patElectrons( new ElectronCollection ) ;
 
   // Fillers for ValueMaps:
-  std::auto_ptr<edm::ValueMap<double> > regrEnergyMap(new edm::ValueMap<double>() );
-  edm::ValueMap<double>::Filler energyFiller(*regrEnergyMap);
+  std::auto_ptr<edm::ValueMap<float> > regrEnergyMap(new edm::ValueMap<float>() );
+  edm::ValueMap<float>::Filler energyFiller(*regrEnergyMap);
 
-  std::auto_ptr<edm::ValueMap<double> > regrEnergyErrorMap(new edm::ValueMap<double>() );
-  edm::ValueMap<double>::Filler energyErrorFiller(*regrEnergyErrorMap);
+  std::auto_ptr<edm::ValueMap<float> > regrEnergyErrorMap(new edm::ValueMap<float>() );
+  edm::ValueMap<float>::Filler energyErrorFiller(*regrEnergyErrorMap);
 
 
   // Reserve the vectors with outputs
-  std::vector<double> energyValues;
-  std::vector<double> energyErrorValues;
+  std::vector<float> energyValues;
+  std::vector<float> energyErrorValues;
   energyValues.reserve(nElectrons_);
   energyErrorValues.reserve(nElectrons_);
 
