diff --git a/RecoEgamma/EgammaTools/src/ggPFPhotons.cc b/RecoEgamma/EgammaTools/src/ggPFPhotons.cc
--- a/RecoEgamma/EgammaTools/src/ggPFPhotons.cc	
+++ b/RecoEgamma/EgammaTools/src/ggPFPhotons.cc	
@@ -473,7 +473,7 @@
   EcalRecHitCollection::const_iterator ee;
   double MaxEta=-99;
   double MaxPhi=-99;
-  double MaxR=-99;
+  //  double MaxR=-99;
   for(;cit!=sc.clustersEnd();++cit){
     std::vector< std::pair<DetId, float> >bcCells=(*cit)->hitsAndFractions();
     DetId seedXtalId = bcCells[0].first ;
