diff --git a/.gitignore b/.gitignore
index 82c501e..0a7523e 100644
--- a/.gitignore
+++ b/.gitignore
@@ -4,3 +4,4 @@ __init__.py
 .#*
 #*#
 *~
+Calibration
\ No newline at end of file
diff --git a/DataFormats/EgammaCandidates/src/GsfElectron.cc b/DataFormats/EgammaCandidates/src/GsfElectron.cc
index 45b977f..84c596a 100644
--- a/DataFormats/EgammaCandidates/src/GsfElectron.cc
+++ b/DataFormats/EgammaCandidates/src/GsfElectron.cc
@@ -81,6 +81,7 @@ GsfElectron::GsfElectron
    //closestCtfTrack_(electron.closestCtfTrack_),
    fiducialFlags_(electron.fiducialFlags_),
    showerShape_(electron.showerShape_),
+   full5x5_showerShape_(electron.full5x5_showerShape_),
    dr03_(electron.dr03_), dr04_(electron.dr04_),
    conversionRejection_(electron.conversionRejection_),
    pfIso_(electron.pfIso_),
